app/src/main/java/com/github/se/eventradar/ui/messages/Messages.kt

```diff
@@ -350,33 +450,6 @@ fun PreviewMessagesScreen() {
             "https://firebasestorage.googleapis.com/v0/b/event-radar-e6a76.appspot.com/o/Profile_Pictures%2FYJP3bYiaGFPqx64CT6kHOpwvXnv1?alt=media&token=5587f942-efc7-4cbf-920c-7f24a76d7ad1",
             "",
             "johndoe")
-      })
-}
-
-@Preview(showSystemUi = true, showBackground = true)
```

Comment:
Was the preview removed for any reason?

Reply:
It was a temporary preview I used that I frankly forgot to remove in my previous PR.